docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -153,29 +153,65 @@ The above query is translated to the following Pinot PQL query::
     WHERE col3 IN('FOO', 'BAR') and col4 > 50
     TOP 30000
 
+.. pinot-type-mapping:
 
+Type mapping
+------------
+
+Because Trino and Pinot each support types that the other does not, this
+connector :ref:`modifies some types <type-mapping-overview>` when reading data.
+
+Pinot type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Data types
-----------
-
-Pinot does not allow null values in any data type and supports the following primitive types:
-
-==========================   ============
-Pinot                        Trino
-==========================   ============
-``INT``                      ``INTEGER``
-``LONG``                     ``BIGINT``
-``FLOAT``                    ``REAL``
-``DOUBLE``                   ``DOUBLE``
-``STRING``                   ``VARCHAR``
-``BYTES``                    ``VARBINARY``
-``JSON``                     ``JSON``
-``INT_ARRAY``                ``VARCHAR``
-``LONG_ARRAY``               ``VARCHAR``
-``FLOAT_ARRAY``              ``VARCHAR``
-``DOUBLE_ARRAY``             ``VARCHAR``
-``STRING_ARRAY``             ``VARCHAR``
-==========================   ============
+The connector maps Pinot types to the corresponding Trino types
+according to the following table:
+
+.. list-table:: Pinot type to Trino type mapping
+  :widths: 30, 30, 20
+  :header-rows: 1
+
+  * - Pinot type
+    - Trino type
+    - Notes
+  * - ``INT``
+    - ``INTEGER``
+    - Pinot does not allow null values in any data type.
```

Comment:
primitive types != any data type.
Also, the note for all types should not exist in `INT` entry. Could you move to before/after this table?

Reply:
Moved the note to underneath the table. Actually, that was the only way I could get the table not to wrap some of the column entries. I've tried endless combinations of column widths to no avail. @jhlodin can you please advise? The original format does not wrap but the list table format has column width issues.

Reply:
Can you share a screenshot of what the wrap looks like?

Reply:
With notes column, widths set to 50, 50, 10

![image](https://user-images.githubusercontent.com/3756245/189961621-501a522a-cd8e-4fa4-848b-6d48935f30de.png)

With empty notes column (as it is is currently published doc) removed set to 50, 50

![image](https://user-images.githubusercontent.com/3756245/189961875-35df7a1c-51cd-4948-8dc4-c319c03bed86.png)

Reply:
Well, a notes column of width 10 is certainly going to wrap weirdly - that's extremely thin. Try setting the notes column to 70 and add the note back in, see how that looks.

Reply:
The notes column is back in, that was just an experiment. I also removed widths altogether since it is supposed to be smart about wrapping. I tried settings similar to other connectors but no go. Setting to 70 didn't help. 

This PR is not ready to merge.

Reply:
I did not add the note back into the table because it isn't applicable to just INT, and Yuya requested I move it to beneath the table. IT was part of the former lead-in before.

Reply:
Discussed offline. Add `:width: 400px` to the table definition properties to resolve this.

Reply:
Specifying width using px doesn't work well in case of smartphone and there're other places of narrowed table.
e.g. https://trino.io/docs/current/connector/kudu.html#behavior-with-schema-emulation

I would recommend removing the workaround and fix all places later once we found better approach.

Reply:
@ebyhr and @jhlodin I removed the workaround as Yuya suggested.

Reply:
cc: @mosabua ^